scripts/e2e/bootstrap_job/spec/machines.template

```diff
@@ -0,0 +1,34 @@
+items:
+- apiVersion: "cluster.k8s.io/v1alpha1"
+  kind: Machine
+  metadata:
+    generateName: clusterapi-prow-
+    labels:
+      set: master
+  spec:
+    providerConfig:
+      value:
+        apiVersion: "vsphereproviderconfig/v1alpha1"
+        kind: "VsphereMachineProviderConfig"
+        vsphereMachine: "standard-master"
+        machineVariables:
```

Comment:
Remove the two above.  machineVariables and vsphereMachine.

Reply:
fixed.